release-notes/release-3.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: February 28, 2024
+---
+
+# Features
+- Metrics Explorer in Settings provides users a single location to view all of their collected metrics -  users can search for metrics by resource name, key, or label names
+- Dynamic filters in the alert module for users based on their existing alert configurations
+- Users can mute their already created Alerts in the details screen
+- Alerts are dynamically changed based on the last triggered alert for that rule
+- Data Ingestion Intervals allow users to control the interval on which they ingest data for any new dashboard
```

Comment:
This is at the widget level and is more about how the metrics are computed. Suggested edit,

"Data Ingestion Intervals allow users to adjust the window at which Widgets compute metrics, e.g. 30s or 1m"